internal/server/handler.go

```diff
@@ -297,3 +297,10 @@ func (s *Server) BulkFindEntities(
 ) (*pb.BulkFindEntitiesResponse, error) {
 	return recon.BulkFindEntities(ctx, in, s.store, s.mapsClient)
 }
+
+// RecognizePlaces implements API for Mixer.RecognizePlaces.
```

Comment:
put this in handler_v1.go since it servers /v1 API?

Reply:
done